### ISSUE TITLE
Microphone Support

### ISSUE THREAD

**Issue by rooly4** [raphael/linux-samus#180]
The built-in microphone has not worked since 4.8, although Pulse detects an input device.

Is there some alsa configuration or channel that I'm missing for this to work?

**Comment by pandeiro** [comment]
I was able to get mine working by running `alsamixer -c 0` then pressing `F4` and raising the 'Mic' value (mine was at 0 initially).

**Comment by rooly4** [comment]
Mine was at 100 when I followed the same steps:

![image](https://cloud.githubusercontent.com/assets/4007907/21439260/95e9321e-c852-11e6-9764-ac5cb5ab1859.png)

**Comment by pandeiro** [comment]
Do you see the levels changing when running `pavucontrol` and looking at the 'Input Devices' tab?

**Comment by rooly4** [comment]
No. alsamixer values have no affect on pavucontrol values it seems, and the bdw-rt5677 mic does not detect input signal in pavu

![image](https://cloud.githubusercontent.com/assets/4007907/21439401/72343584-c853-11e6-88ea-c56fca75591d.png)

**Comment by pandeiro** [comment]
Odd, yeah my panel shows _two_ entries for bdwrt5677.

![image](https://cloud.githubusercontent.com/assets/361596/21439484/78c4aba0-c875-11e6-8fde-d92058ff6ce1.png)

I imagine our systems differ significantly though because I'm running Arch w/o any desktop environment (but with pulseaudio). Wish I could help more -- microphone issues plagued me for a long time with this machine!

**Comment by vrcod** [comment]
I am running Ubuntu 16.10 too and I can't get the mic to work. No amount of fiddling with alsamixer seems to help.

**Comment by Pdbrantley** [comment]
Wanted to bump this thread. I'm having the same issue. Is there any addition info I could provide that might help find a solution?

**Comment by rooly4** [comment]
I actually managed to get the Mic working after a reinstall. Make sure you have /etc/modprobe.d/sound.conf in place, reboot, then run the audio scripts (several times to be sure).

If you've migrated from a kernel before 4.8 then I'm not sure what will make it work.
